### ISSUE TITLE
Android app stops image-taking mode

### ISSUE THREAD

**Issue by jfburke3** [mapillary/mapillary_issues#2511]
### Basic information
* Platform: Android 6.0.1
* Version: Mapillary 3.39
* System/Browser: Samsung Galaxy S7 Edge

### Steps to reproduce behavior

 1.  Start shooting a sequence.
 2.  Periodically check to see if it's still actively shooting the sequence, or has stopped image-taking mode (indicated by the start/stop button appearing as a round button instead of a square).

### Expected behavior

The app continues to take pictures until I press the Stop button.

### Actual behavior

Intermittently, the app simply stopped itself after several dozens (over 100?) images taken in a sequence.  (Meaning that yes, it did often continue to shoot a sequence until I pressed stop, but occasionally, it seems to have pressed stop by itself.)

### Additional information

This may or may not be the same problem as described in #2449 .  Where the two differ is that in that one, the start/stop button remained in "press me to stop" mode (i.e., the app seemed to think it was still taking pictures).  With today's issue, the button was clearly in "press me to start" mode (i.e., it was round, not square).  Also, in that issue, it would stop very very early in the sequence, usually just one or two pictures in.  In this one, it shot over a hundred before stopping.  For this reason, I'm submitting this as a separate case, although if you think they're the same, feel free to reference & close this one.

Also, the symptoms are different than #2450 .  In that one, the screen freezes or goes black for a period of time, then (depending on which Mapillary release I was using), either displays an error message or exits to the main screen, and re-entering camera mode displays an error message.  Here, it simply stops the sequence, but the camera remains "live" (e.g., I can see the camera's view changing with my forward motion).

I would e-mail trace logs, but apparently when I used the "send debug logs" function on the app menu and then saved the e-mail on my phone as draft so I could compose a body later, it didn't retain the attachment, and I didn't realize that until after I'd cleared the logs later.  Never have seen that happen before.  (It's possible that the "attachment" was just a link to the log file, and wasn't an actual copy of the log file, so when I cleared the logs, it deleted what would have been attached.  I'll try to remember that for the future.)  I'll try to reproduce this again, and just send the logs right then instead of saving a draft of the e-mail.

**Comment by katrinhumal** [comment]
Thanks for a detailed report @jfburke3! Your description is a real role model of how to report bugs. We just released 3.41, can you please check if this keeps happening? And as you already marked, logs would be highly useful (too bad it went like that when you wanted to save the draft). Thanks in advance!

**Comment by smsm1** [comment]
I've been using the Android Maillary app to capture photos using the bike mount on my Fairphone 2, and having been having the same issue where the capturing stops. I tried both time based and distance based auto modes. I also found some sequences would produce just green images, which I deleted before upload. 

I've not been able to figure out a pattern to when the app stops taking photos, but it was so unreliable yesterday I just gave up, and wished I had the Garmin virb and it's mount with me.

I'm on version 3.45.

**Comment by jfburke3** [comment]
It happened again several on Sunday with version 3.48.  I didn't get trace logs at the time, unfortunately.

**Comment by katrinhumal** [comment]
@smsm1 and @jfburke3 sorry for the lack of updates here. What about the versions 3.50 and above, during the last week? Please turn logs on just in case. Btw even if you have the `Save debug logs` option off, you can still send us logs! If you do this close enough after the problem occurred, then we should still be able to read out some useful stuff from there. Better than none, as I've been told in any case :)

**Comment by voschix** [comment]
I keep having both variants of the problems with 3.50 and above. There are so many problems at the same time, it's difficult to keep track of them. I certainly have seen spontaneous stopping with camera picture frozen for a short while. Consistently I was not able to restart picture taking by pressing the round red button. I consistently needed to exit/re-enter the Mapillary App. Often picture taking would start again after this, but also, often, it would not. In those case I repeated the exit/restart App operation and waited a while, and then, most of the time, picture taking would restart. My gut feeling is that the app has become more resourec-greedy or that Android has become more resource-greedy or a combination of both. With other words, I would need to strip my phone of all other apps and try the Mapillary app on its own.

**Comment by jfburke3** [comment]
This might have been fixed by 3.67. This strongly resembles the problem I was having when I e-mailed in logs when @peterneubauer was on vacation a couple of weeks ago. I went on a long drive Sunday afternoon through the mountains, and used distance-capture mode the entire time. I did not experience the problem. (Not prepared to say I'm certain it's fixed, because it was generally intermittent, but it's looking good.)
